src/common/module/cloudsc_field_state_mod.F90

```diff
@@ -388,7 +396,7 @@ SUBROUTINE CLOUDSC_FIELD_STATE_LOAD(SELF, NPROMA, NGPTOT, NGPTOTG, USE_PACKED)
       ! Allocate bulk buffers for output 3D fields
       NFIELDS = 14
       ! CALL FIELD_INIT(SELF%DATA_RWONLY, NPROMA, SELF%KLEV+1, NFIELDS, SELF%NBLOCKS)
-      SELF%DATA_RWONLY => malloc_host_pinned_4d([NPROMA, SELF%KLEV+1, NFIELDS], SELF%NBLOCKS)
+      SELF%DATA_RWONLY => MALLOC_HOST_PINNED_4D([NPROMA, SELF%KLEV+1, NFIELDS], SELF%NBLOCKS)
```

Comment:
[No action needed] In future versions, we might want to think about encapsulating this ins a "field set" object or similar to provide better encapsulation. But this requires development in the FIELD API, and for now this is fine here.